### PR TITLE
Begin choral implementation

### PR DESCRIPTION
**Pull request by andreirtaylor**
Not complete as this does not respect the timeout set in the choral. But this is the beginnings of setting up chorals in the application.

Features implemented

function structure
- first parameter is a object containing the names of each of the child chroral as members
- second parameter is a callback which takes in a single parameter which is the return value of the data.

### CODE REVIEW

**Comment by andreirtaylor** [comment]
#15

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ You will need to create a new config file based on the template:
 
 Now overwrite the config file with the correct clientID and clientSecret (you will need to get them from google).
 
+run the command 
+
+`npm install`
```
Comment (andreirtaylor) [inline README.md:15]: This is now necessary as it is removed from the docker compose file.

Comment (cpjk) [reply README.md:15]: def forgot to run that

**Review by cpjk — APPROVED** [review]
lgtm

**Comment by cpjk** [comment]
needs rebase

**Comment by andreirtaylor** [comment]
Rebased

**Pull request merged**